### PR TITLE
Add config keys from the appconfig table for core

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/owncloud/configreport/issues/22

To be backported to stable9.1 and stable9.

### CODE REVIEW

File: lib/ReportDataCollector.php [hunk 1]
```diff
@@ -203,20 +204,41 @@ private function getSystemConfigDetailArray() {
...
+		return $this->sanitizeValues($appConfig);
+	}
+	/**
 	 * @return array
 	 */
 	private function getAppsDetailArray() {
 		// Get app data
 		foreach($this->apps as &$app) {
-			if($app['active']) {
+			if($app['active'] || $app['id'] === 'core') {
```
Comment (phisch) [inline lib/ReportDataCollector.php]: Doesn't that get rid of the app configs?

Comment (PVince81) [inline lib/ReportDataCollector.php]: it doesn't, but thanks for pointing at this. it was my previous attempt that didn't work.

will remove

**Comment by PVince81** [comment]
adjusted (squashed)

**Comment by PVince81** [comment]
will backport later once the RCs are out and stable branches are open for merging again

**Comment by PVince81** [comment]
@phisch can you backport ?

**Comment by phisch** [comment]
Was merged into master before the stable10 branch was opened, so no backport needed.

**Pull request merged**